package.json

```diff
@@ -54,12 +54,12 @@
         "fetch-mock": "^9.11.0",
         "gamp": "0.2.1",
         "glob": "^8.1.0",
-        "handlebars": "1.3.0",
+        "handlebars": "^4.7.8",
         "interactjs": "1.3.4",
-        "jquery": "1.9.1",
+        "jquery": "^3.7.1",
         "jquery-mockjax": "^2.6.0",
         "jquery-simulate": "^1.0.2",
-        "lodash": "2.4.1",
+        "lodash": "^4.17.21",
```

Comment:
I'm not sure we can get away with casually bumping all these libs.

The reason is that `tao-core-sdk` is not used in isolation, it is used as part of a fully-installed TAO where several other components are still using handlebars 1.3.0, jquery 1.9.1 and lodash 2.4.1. In some cases we would have a conflict in the browser (jquery for sure / lodash not sure).

That's why we intend to handle the upgrade of these libs with a dedicated task designed to upgrade each one in all places at the same time.
- Handlebars: https://oat-sa.atlassian.net/browse/TR-3264
- Jquery: https://oat-sa.atlassian.net/browse/TR-3262
- Lodash: task doesn't exist, but perhaps it should

The overall initiative to move some lodash usages to the v4 API is good, though. If you want to concentrate on that, you can continue, but be sure to test this library installed in tao-core in a full TAO Community.

Reply:
I realized that this one is indeed to risky. I guess I'll go with `npm audit fix` without `--force` for now.

Reply:
BTW, as Martin explained, you should have a look at how the FE packages relate to each other on TAO.

We have a bunch of libraries that are dependent on each other, hierarchically. When updating one, you need to spread the change across all others. 

As a quick overview, we have in order:
- tao-core-shared-libs
- tao-core-libs
- tao-core-sdk
- tao-core-ui
- tao-item-runner
- tao-item-runner-qti
- tao-test-runner
- tao-test-runner-qti

And besides that, a galaxy of smaller packages that are consumed for the tooling, like:
- grunt-tao-bundle
- browserslist-config-tao
- eslint-config-tao
- prettier-config
- etc.

And some other specific libraries...

In TAO, we also have extensions that will contain frontend code too, consuming and relying on the above-mentioned libs.

A change in lodash, will impact all of that. The same with jQuery, Handlebars, Select2...